AuxTel/AuxTel-Standard-Operations/AuxTel-Daytime-Operations/Daytime-Checkout.rst

```diff
@@ -95,12 +100,9 @@ It will leave the telescope in position to start taking afternoon calibrations.
 If you are not sure how you want to leave the telescope,
 consider running this script as it will further exercise the system. 
 
-Other options include: prepare_for/onsky, prepare_for/vent and shutdown_all
+Other options include: :file:`prepare_for/onsky.py`, :file:`prepare_for/vent.py` or :file:`shutdown_all.py`
 
 Quick Troubleshooting
 =====================
 
-Add some tips here on what to do if things don't work. 
```

Comment:
**Maybe something like this:** If any of the checkout scripts fails, consider change the ATTCS state to \`\`STANDBY\`\` and then \`\`ENABLE\`\`. Relaunch the failed script.

Reply:
I will remove the troubleshooting section for a later review.